spec/model/github/github_repository_spec.rb

```diff
@@ -13,25 +13,44 @@
     allow(Aws::S3::Client).to receive(:new).and_return(blob_storage_client)
   end
 
-  it ".destroy_blob_storage" do
-    expect(blob_storage_client).to receive(:delete_bucket).with(bucket: "gph0hh0ahdbj6ng2cc3rvdecr8")
-    expect(cloudflare_client).to receive(:delete_token).with(github_repository.access_key)
-    expect(github_repository).to receive(:update).with(access_key: nil, secret_key: nil)
-    github_repository.destroy_blob_storage
+  describe ".destroy_blob_storage" do
+    it "deletes the bucket and token" do
+      expect(blob_storage_client).to receive(:delete_bucket).with(bucket: "gph0hh0ahdbj6ng2cc3rvdecr8")
+      expect(cloudflare_client).to receive(:delete_token).with(github_repository.access_key)
+      expect(github_repository).to receive(:update).with(access_key: nil, secret_key: nil)
+      github_repository.destroy_blob_storage
+    end
+
+    it "runs even if the bucket already deleted" do
```

Comment:
maybe: succeeds if the bucket is already deleted

Reply:
Thanks. Changed